### PR TITLE
Remove pytest fixture prefix snd name arg in decorator

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

Remove pytest fixture prefix snd name arg in decorator

### Changed

- Remove pytest fixture prefix snd name arg in decorator

### Fixed

-

### How to prepare for test

- [ ] ssh to hasta (depending on type of change)
- [ ] install on stage:
```bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b YOURBRANCH```

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/cli/add/conftest.py [hunk 1]
```diff
@@ -6,13 +6,13 @@
 import pytest
 
 
-@pytest.fixture(name="file_data")
-def fixture_file_data(second_sample_vcf: Path, case_id: str, sample_tag_names: List[str]) -> dict:
+@pytest.fixture()
```
Comment (seallard) [inline tests/cli/add/conftest.py:9]: I think `@pytest.fixture` is enough, the parentheses are not required when no arguments are given

**Review by seallard — APPROVED** [review]
Looks good 👍

File: tests/cli/add/conftest.py (hunk 1 above)
Comment (henrikstranneheim) [reply tests/cli/add/conftest.py]: Good suggestion - fixed

**Comment by henrikstranneheim** [comment]
Nothing to deploy

**Pull request merged**